components/studio/bin/hab-studio.sh

```diff
@@ -1063,11 +947,9 @@ report_env_vars() {
     info "Exported: no_proxy=$no_proxy"
   fi
 
-  if [ "$(load_secrets)" ]; then
-    for secret in $(load_secrets); do
-      info "$(echo $secret | $bb awk -F= '{print "Exported:",$1"=[redacted]"}')"
-    done
-  fi
+  for secret_name in $(load_secrets | $bb cut -d = -f 1); do
+    info "Exported: $secret_name=[redacted]"
+  done
```

Comment:
Also interested in your input here, @elliott-davis. I think this is a bit simpler without sacrificing safety.

Reply:
:+1: - Plus it's one less call to awk